cds-snc/cppd-medical-report
# use routePath helper for routes

# Description

Previous to the work done in #108 there was no way to reference routes with parameters by name, so we had the urls hardcoded in, which [caused problems](https://github.com/cds-snc/cppd-medical-report/pull/112#issuecomment-561725990) when navigating the edit/delete functions on french pages.

## Testing

In French:
- Create some conditions
- Try editing or deleting them
- Create some medications
- Try editing or deleting them
- Create some treatments
- Try editing or deleting them
You should remain on the french interface the whole time.
